api-svc/raid-api/src/main/java/au/org/raid/api/repository/SubjectTypeSchemaRepository.java

```diff
@@ -16,7 +16,8 @@ public class SubjectTypeSchemaRepository {
 
     public Optional<SubjectTypeSchemaRecord> findById(final Integer id) {
         return dslContext.selectFrom(SUBJECT_TYPE_SCHEMA)
-                .where(SUBJECT_TYPE_SCHEMA.ID.eq(id))
+//                .where(SUBJECT_TYPE_SCHEMA.ID.eq(id))
+                .where(SUBJECT_TYPE_SCHEMA.ID.eq(3))
```

Comment:
What's this?

Reply:
Used for debugging - forgot to take it out, it's fixed now